blocks/highlight/highlight.js

```diff
@@ -1,17 +1,28 @@
 export default async function decorate(block) {
-  let spotlight = document.createElement('div');
-  spotlight.classList.add('spotlight');
-  const others = document.createElement('div');
-  others.classList.add('others');
-  block.querySelectorAll(':scope > div').forEach((div, index) => {
-    if (index === 0) {
-      spotlight = div;
-    } else {
+  if (block.classList.contains('feed-newsroom')) {
```

Comment:
Looks like there's code repetition here in `if/else` . You can do as follows:
1. Add a `no-spotlight` variant of `highlight` block.
2. If that variant is set on the block, skip the `spotlight` div addition.
3. Create the `others` div similarly in all cases.

Reply:
Investigating further on this - Now our newsroom-feed block JS is calling a block - highlight which is then further passed to buildBlock function where highlight js will be called. What you are mentioning - Add a no-spotlight variant of highlight block - how ? when we are not mentioning any highlight block in the word document. Its there in the JS file.

Reply:
It can be set on here
![image](https://github.com/hlxsites/sunstar/assets/1191451/6962b9bf-40cb-4c2d-bb26-ffff7efd3048)